Dockerfile

```diff
@@ -52,26 +54,8 @@ ENV         VIRTUAL_ENV=/opt/venv \
 RUN         apk --update add \
               openssh-client~=9.1
 RUN         source $VIRTUAL_ENV/bin/activate
-COPY        .flake8 ./.flake8
-COPY        .git/ ./.git
-COPY        .darglint ./.darglint
-COPY        canu/ ./canu
-COPY        docs/ ./docs
-COPY        network_modeling/ ./network_modeling
-COPY        pyinstaller_hooks/ ./pyinstaller_hooks
-COPY        tests/ ./tests
-COPY        canu.spec ./canu.spec
-COPY        canuctl ./canuctl
-COPY        Dockerfile ./Dockerfile
-COPY        entry_points.ini ./entry_points.ini
-COPY        LICENSE ./LICENSE
-COPY        Makefile ./Makefile
-COPY        MANIFEST.in ./MANIFEST.in
-COPY        mkdocs.yml ./mkdocs.yml
-COPY        noxfile.py ./noxfile.py
-COPY        pyinstaller.py ./pyinstaller.py
-COPY        pyproject.toml ./pyproject.toml
-RUN         python -m pip install --editable .[ci,docs]
+COPY        . ./
```

Comment:
This was intentionally broken out into the specific files to prevent the injection of unknown or malicious files--a quality gate caught by sonar cloud.  I'm not sure why it doesn't complain about it now

Reply:
Gotcha.

I was staring at it for a while, and only saw a handful of files missing. The several `COPY` steps introduce a new layer for each `COPY`, so in general I try to keep `Dockerfile` calls to a minimum. I wasn't sure if the stage build makes that layer point moot or not, but I don't need to die on the hill and will put it back if you'd prefer.

Reply:
Yes, I do like multiple `COPY`s for the security aspect.  The file is already quite long and the multiple stages makes it not matter as much.  Thanks for changing it back